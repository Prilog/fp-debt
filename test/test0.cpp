int fib(int x) {
	if (x < 2) {
		return 1;
	} else {
		return fib(x - 1) + fib(x - 2);
	}
}

int main() {
	int times = 6;
	int result = fib(times);
	cout << result;
	return 0;
}