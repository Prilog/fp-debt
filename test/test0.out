=================
print required action:
"exe" - execute file CDSLResult.hs
*filename* - interpretes CDSLResult.hs into file *filename*
13main returned with exit code: 0
